src/ConnectedMode/Install/ImportBeforeInstallTrigger.cs

```diff
@@ -60,7 +55,7 @@ private void OnPreSolutionBindingChanged(object sender, ActiveSolutionBindingEve
 
         private void OnPreSolutionBindingUpdated(object sender, EventArgs e) => TriggerUpdateAsync().Forget();
 
-        private async Task TriggerUpdateAsync()
+        internal async Task TriggerUpdateAsync()
```

Comment:
By convention, this should be `public` rather than `internal` since it is part of the public API that we expect other product classes to use.

We use `internal` on methods/properties to indicate the method/property should only be used by the declaring type and from test code.

Reply:
Ah good catch, thank you!